<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0,maximum-scale=1">
		
		<title>Li Lab</title>

		<!-- Loading third party fonts -->
		<link href="http://fonts.googleapis.com/css?family=Roboto:300,400,700|" rel="stylesheet" type="text/css">
		<link href="fonts/font-awesome.min.css" rel="stylesheet" type="text/css">

		<!-- Loading main css file -->
		<link rel="stylesheet" href="style.css">
		
		<!--[if lt IE 9]>
		<script src="js/ie-support/html5.js"></script>
		<script src="js/ie-support/respond.js"></script>
		<![endif]-->
		<style>
			element {
    width: 100%;
}
table {
    background-color: transparent;
}
table {
    border-collapse: collapse;
    border-spacing: 0;
}
 /* Style the button that is used to open and close the collapsible content */
 .collapsible {
  background-color: #eee;
  color: #444;
  cursor: pointer;
  padding: 18px;
  width: 100%;
  border: none;
  text-align: left;
  outline: none;
  font-size: 15px;
}

/* Add a background color to the button if it is clicked on (add the .active class with JS), and when you move the mouse over it (hover) */
.active, .collapsible:hover {
  background-color: #ccc;
}

/* Style the collapsible content. Note: hidden by default */
.content {
  padding: 0 18px;
  display: none;
  overflow: hidden;
  background-color: #f1f1f1;
} 
.btn {
  background-color: DodgerBlue;
  border: none;
  color: white;
  padding: 12px 30px;
  cursor: pointer;
  font-size: 20px;
}

/* Darker background on mouse-over */
.btn:hover {
  background-color: RoyalBlue;
}
		</style>

	</head>


	<body>
		
		<div class="site-content">
			<header class="site-header" data-bg-image="">
				<div class="container">
					<div class="header-bar">
						<a href="index.html" class="branding">
							<img src="images/li.ico" alt="" class="logo">
							<div class="logo-type">
								<h1 class="site-title">Dr Li's Lab</h1>
								<small class="site-description">Fruit Molecular Biology Lab</small>
							</div>
						</a>

						<nav class="main-navigation">
							<button class="menu-toggle"><i class="fa fa-bars"></i></button>
							<ul class="menu">
								<li class="menu-item"><a href="index.html">Home</a></li>
								<li class="menu-item"><a href="about.html">About</a></li>
								<li class="menu-item"><a href="services.html">Research Overview</a></li>
								<li class="menu-item current-menu-item"><a href="projects.html">Our projects</a></li>
								<li class="menu-item"><a href="resources.html">Resources</a></li>
								<li class="menu-item"><a href="contact.html">Contact</a></li>
							</ul>
						</nav>

						<div class="mobile-navigation"></div>
					</div>
				</div>
			</header>

			<br>
			<div class="page-head" data-bg-color="#FED106">
				<div class="container">
				<h9>Grain Size Project</h9>
					
				</div>
			</div>
<br>

			<main class="main-content">
				<div class="fullwidth-block">
					<div class="container">
						<a href="projects.html" class="button back"><img src="images/arrow-back.png" alt="">Back to projects</a>
						<div class="row">
							<div class="col-md-6">
								<figure>
									<img src="images/figure.jpg" alt="">
								</figure>
							</div>
							<div class="col-md-6">
								<h2 class="section-title">CRISPR-based genome editing of grain size regulators for novel variation to increase wheat genetic yield potential</h2>
								<ul class="project-info">
									<li><strong>Founded On:</strong> 06/10/14</li>
									<li><strong>Funding Agency:</strong> USDA</li>
									<li><strong>Project Director:</strong>  Wanlong Li Institution: South Dakota State University</li>
									<li><strong>Co-Project Director::</strong>    </li>
									<li><strong>Project Members:</strong> Lei Hua, Ajay Gupta</li>
									
								</ul>
							</div>
						</div>

						<div >
							<div >
								<p>Grain size (GS) is the major driver for further growth of wheat yield. Much progress has been made in dissecting the genetic network regulating GS in the model plants rice and Arabidopsis, wherein more than half of the underlying genes are negative GS regulators or negatively regulated by microRNAs. But little is known about GS regulation in wheat. This knowledge gap significantly limits our effort to improve wheat yield. Hypothesizing that conserved genetic pathways underlie GS variation in rice and wheat, we teamed up with expertise in wheat genetics/genomics and genome editing to address Area Priority 1 of this NIFA-IWYP program (A1142) by creating novel variation of the negative GS regulators using CRISPR-based technology for significant increase of genetic yield potential. </p>
							
								
								<hr><button type="button" class="collapsible"><center>Our Objectives</center></button><div class="content">
								<hr>
								<ol>
									&emsp;<li>Develop an improved CRISPR/Cas9 system for editing GS candidate genes;</li>
									&emsp;<li>Identify mutations in the GS candidate genes;</li>
									&emsp;<li>Characterize the effect of mutations on GS; and</li>
									&emsp;<li>Transfer beneficial mutations into elite durum wheat.</li>
								</ol>
								<br>
							</div>
							<hr>
								<p>We have optimized the CRISPR/Cas9 system for high-efficiency genome editing in wheat, identified 32 GS candidate genes in wheat genome, and set up platforms for high-throughput GS phenotyping and for early-stage mutation detection. We expect to develop an improved CRIPSR/Cas9 system for wheat with enhanced editing efficiency, targeting flexibility and accuracy, a panel of novel GS mutations and associated knowledge, a package of novel germplasm with enhanced grain yield potential and functional markers. Thus, this project will significantly contribute to IWYPâ€™s goal to increase wheat yield by 50% by 2034.</p>
							</div>

							<div >
								<br>
								    
								
								<hr><button type="button" class="collapsible"><center>Primers used in this study</center></button><div class="content"><br> <center><table>
  <tbody><tr>
    <th>Gene name</th>
    <th>Primer name (F/R)</th> 
    <th>Forward primer (5`-&gt; 3`)</th>
    <th>Reverse primer (5`-&gt; 3`)</th>
  </tr>
 <tr><td><a href="seqData/APG_1AS.txt">APG-1AS</a></td><td>WL3971/WL3972</td><td>GAGTTGGTGCGTGCGCTA</td><td>GGACTCGACGGACTTGCTT</td></tr>
<tr><td><a href="seqData/APG_1BS.txt">APG-1BS</a></td><td>WL3973/WL3974</td><td>CCGTGCTCTTCCTTTTCTTC</td><td>ACCATGTGTCCAGCGCTTT</td></tr>
<tr><td><a href="seqData/APG_1DS.txt">APG-1DS</a></td><td>WL4092/WL4093</td><td>TGGTGCGCTACTCCTCCT</td><td>GGTTTCTTGCGAGGGTTGAC</td></tr>
<tr><td><a href="seqData/BAS1-3_3A.txt">BAS1-3A</a></td><td>WL4234/WL4276</td><td>CACAGCTCAACGAGTGAGTGAGT</td><td>CTGGGGACGTATGTATGGATGT</td></tr>
<tr><td><a href="seqData/BAS1-3_3B.txt">BAS1-3B</a></td><td>WL4236/WL4237</td><td>CTATCACAGGACGCCAGG</td><td>CGAAGAAGAGACAACTCCAAGGT</td></tr>
<tr><td><a href="seqData/BAS1-3_3DS.txt">BAS1-3DS</a></td><td>WL4238/WL4239</td><td>CTCTGTAGCACGCCGCAA</td><td>TGGGAGATCAAGCAAACAAA</td></tr>
<tr><td><a href="seqData/CKX2-3.1_3AS.txt">CKX2-3.1-3AS</a></td><td>WL3901/WL3902</td><td>CAACGCACGGACACTTAGC</td><td>CGTGTGCGTACGTAGAAGATGAG</td></tr>
<tr><td><a href="seqData/CKX2-3.1_3BS.txt">CKX2-3.1-3BS</a></td><td>WL4101/WL3904</td><td>TCCTGTGCTGTGTACCAGAG</td><td>ATGAACGTGATTCCACGCC</td></tr>
<tr><td><a href="seqData/CKX2-3.1_3DS.txt">CKX2-3.1-3DS</a></td><td>WL3905/WL3906</td><td>CATTTCATTTATCATGACTTCCC</td><td>GTGCGTGCTTGCCCATATC</td></tr>
<tr><td><a href="seqData/CKX2-3.2_3BS.txt">CKX2-3.2-3BS</a></td><td>WL4082/WL3908</td><td>CTTTGTGTGTGTAATTTCTCCCG</td><td>GCTCTAATTAAATACGAAGCGTG</td></tr>
<tr><td><a href="seqData/CKX2-3.2_3DS.txt">CKX2-3.2-3DS</a></td><td>WL3909/WL3910</td><td>AAGAATCTTTCGTCGGTCCT</td><td>GGTGAACGAGTTGTGGTTACTTA</td></tr>
<tr><td><a href="seqData/CKX2-3.3_3AS.txt">CKX2-3.3-3AS</a></td><td>WL3911/WL3912</td><td>TACCTACGCAAGCAGGATGGA</td><td>GACGAAGACGACGATGACG</td></tr>
<tr><td><a href="seqData/CKX2-3.3_3BS.txt">CKX2-3.3-3BS</a></td><td>WL3913/WL3914</td><td>TTGCAAGCCCCAATAACG</td><td>ACAGCGTCGAACAGGTCA</td></tr>
<tr><td><a href="seqData/CKX2-3.3_3DS.txt">CKX2-3.3-3DS</a></td><td>WL3915/WL3916</td><td>GGGTCCCATGTAGTTTACCTTTA</td><td>TAGGCACCATAGGCACGG</td></tr>
<tr><td><a href="seqData/CKX2-3.5_3AS.txt">CKX2-3.5-3AS</a></td><td>WL3919/WL3920</td><td>AAGACTCACAGGAAGCAAGCTAA</td><td>CGCAAACAAAAGATGATACTCAC</td></tr>
<tr><td><a href="seqData/CKX2-3.5_3BS.txt">CKX2-3.5-3BS</a></td><td>WL3921/ WL3922</td><td>ACCATCCGAAAGAAAACGAA</td><td>GCATTAGTACGTGAAGTCGTGAA</td></tr>
<tr><td><a href="seqData/CKX2-3.5_3DS.txt">CKX2-3.5-3DS</a></td><td>WL4056/WL4057</td><td>CGAACCAGATGGATCCGAG</td><td>CCATTAGTACGTGAACGTTGGC</td></tr>
<tr><td><a href="seqData/DEP1_5AL.txt">DEP1-5AL</a></td><td>WL4063/WL4064</td><td>CGTCCGCCCTTGATTTTC</td><td>GACCACTAGCCTAGATGCAGGAG</td></tr>
<tr><td><a href="seqData/DEP1_5BL.txt">DEP1-5BL</a></td><td>WL4003/WL4004</td><td>TGCACTCGCACTCTCTCTCTAAA</td><td>CAGCACCAACAACTGCTACTACT</td></tr>
<tr><td><a href="seqData/DEP1_5DL.txt">DEP1-5DL</a></td><td>WL4143/WL4144</td><td>GCATTTGTTTCCTTGGCAC</td><td>GCTGCCGAGATAAATAACCCTA</td></tr>
<tr><td><a href="seqData/EOD1_4AS.txt">EOD1-4AS</a></td><td>WL4161/WL4162</td><td>TGACAAAAGGACCACTTCTGCT</td><td>CAAACACACTAAAGCACGTCA</td></tr>
<tr><td><a href="seqData/EOD1_4BL.txt">EOD1-4BL</a></td><td>WL4163/WL4164</td><td>GTTCCATAGCCGAGGCCC</td><td>TGATGTGCTGGTGAGCTTTC</td></tr>
<tr><td><a href="seqData/EOD1_4DL.txt">EOD1-4DL</a></td><td>WL4165/WL4166</td><td>TTAGAAGAGAGAACCTTTTTAGGAC</td><td>AAAGCCACAAGCCACGTAAT</td></tr>
<tr><td><a href="seqData/GL3_5AS.txt">GL3-5AS</a></td><td>WL4062/WL3948</td><td>ATCGTTTGTGGTTGACCTTTC</td><td>AGCTAGAAAGTACTGCCGCC</td></tr>
<tr><td><a href="seqData/GL3_5DS.txt">GL3-5DS</a></td><td>WL4066/WL3952</td><td>AGAACTGCCCATGTGACAA</td><td>AAGGTGAGCAGACTTAGACCGT</td></tr>
<tr><td><a href="seqData/GLW7_2AS.txt">GLW7-2AS</a></td><td>WL3935/WL3936</td><td>TATCAATGTTGTGCCTGTTTCT</td><td>CCAAGTGAAAACCAAATGCTC</td></tr>
<tr><td><a href="seqData/GLW7_2BS.txt">GLW7-2BS</a></td><td>WL4047/WL4048</td><td>GCAAGATAAAAACTGGAGG</td><td>GGCGATAAGGTTTATAAGG</td></tr>
<tr><td><a href="seqData/GS2_6AL.txt">GS2-6AL</a></td><td>WL4069/WL3960</td><td>GATTAATCCTGCAAATCTAGCTG</td><td>AAAGACAAGGGCTACTGTGC</td></tr>
<tr><td><a href="seqData/GS2_6BL.txt">GS2-6BL</a></td><td>WL3961/WL3962</td><td>TAGCAGCGTCCCTCCTCCGAA</td><td>AGGGCTACTGTATGGCATGG</td></tr>
<tr><td><a href="seqData/GS2_6DL.txt">GS2-6DL</a></td><td>WL3963/WL3964</td><td>CCTCCTTGTTTGTTTTGCCG</td><td>CATCTTGGCTTGGCTGTCTT</td></tr>
<tr><td><a href="seqData/GW2_6AS.txt">GW2-6AS</a></td><td>WL4007/WL4008</td><td>CGTGTCACAAAACTAATTGGG</td><td>CTACGGCAGAACAAATGCAA</td></tr>
<tr><td><a href="seqData/GW2_6BS.txt">GW2-6BS</a></td><td>WL4009/WL4010</td><td>GGGCCAGCAGCAGAGAGA</td><td>CAAACACAGGCACCTAGCAC</td></tr>
<tr><td><a href="seqData/GW2_6DS.txt">GW2-6DS</a></td><td>WL4011/WL4012</td><td>GACATCATACAAGTGGGGAAGGA</td><td>TACGGCAGAACAAATGCAAC</td></tr>
<tr><td><a href="seqData/GW7-2DS.txt">GW7-2DS</a></td><td>WL3977/WL4046</td><td>GCATGTTCTATCTATGAGCGAGT</td><td>CGACCTAAACTTTGGGAATCTAC</td></tr>
<tr><td><a href="seqData/GW7_2AS.txt">GW7-2AS</a></td><td>WL3979/WL3980</td><td>CTAGCTTCTTGCCCGCAG</td><td>GTGGTTGTTTCTTACTGGACCTT</td></tr>
<tr><td><a href="seqData/GW7_2BS.txt">GW7-2BS</a></td><td>WL3981/WL3982</td><td>CCGTCCTCTCCTCTCCTCT</td><td>ACAAGGGAAAAACCACTTGATA</td></tr>
<tr><td><a href="seqData/GW8_5AL.txt">GW8-5AL</a></td><td>WL3953/WL3954</td><td>CTCGCATCATAAGAATGGAAG</td><td>TGGCAGTTCATCTCGTTGTC</td></tr>
<tr><td><a href="seqData/GW8_5BL.txt">GW8-5BL</a></td><td>WL3955/WL3956</td><td>AAATTCAGGGACAAGGTTCG</td><td>TGCGGTTATTCATTGCTTTC</td></tr>
<tr><td><a href="seqData/GW8_5DL.txt">GW8-5DL</a></td><td>WL3957/WL3958</td><td>GACGTACTGTTCTTCAGTAGCC</td><td>TCATCTCGTTGTCGTTGGAG</td></tr>
<tr><td><a href="seqData/IPA1_7AS.txt">IPA1-AS</a></td><td>WL3965/WL3966</td><td>AGGAGGGTTGCACCAGTTG</td><td>GCGCAGACTTAGGGTAACGTAG</td></tr>
<tr><td><a href="seqData/IPA1_7DS.txt">IPA1-7DS</a></td><td>WL3969/WL3970</td><td>TCTTGCTCTCACCGATCACTAC</td><td>CCGATACTTCAAATTCATGGTT</td></tr>
<tr><td><a href="seqData/LP1_3A.txt">LP1-3A</a></td><td>WL4051/WL4052</td><td>TAGGAGAATGAGCTGTTTTGAG</td><td>CAATGTCCAGCCGAGGCT</td></tr>
<tr><td><a href="seqData/LP1_3B.txt">LP1-3B</a></td><td>WL3943/WL3944</td><td>GGACGGAGGGAGTATTTGAC</td><td>CAAGTGAGAGAACATGGCACT</td></tr>
<tr><td><a href="seqData/LP1_3D.txt">LP1-3D</a></td><td>WL4104/WL3946</td><td>TAGGAGAATGAGCTGTTTCG</td><td>ACTGGTTTTGTTTGATTCCG</td></tr>
<tr><td><a href="seqData/TGW6_3.1_3AL.txt">TGW6-3.1-3AL</a></td><td>WL4053/WL3984</td><td>CTCCATGAGCCGGTTGGT</td><td>TGGACGGACGAATCCACTACTA</td></tr>
<tr><td><a href="seqData/TGW6_3.1_3BL.txt">TGW6-3.1-3BL</a></td><td>WL3985/WL3986</td><td>CGGGCCCAAACACTTACC</td><td>CGATCCAGTGCCTTATCAAC</td></tr>
<tr><td><a href="seqData/TGW6_3.1_3DL.txt">TGW6-3.1-3DL</a></td><td>WL3987/WL3988</td><td>AAGGAAATCCGTCGGGGA</td><td>TCTTTCCATCACCTCAGCG</td></tr>
<tr><td><a href="seqData/TGW6_3.3_3AL.txt">TGW6-3.3-3AL</a></td><td>WL3991/WL3992</td><td>AAACCCACTCGGCGTCAT</td><td>AATGCCTTAGTAGCTTACATGGG</td></tr>
<tr><td><a href="seqData/TGW6_3.3_3BL.txt">TGW6-3.3-3BL</a></td><td>WL4099/WL3994</td><td>GGATCAGTGCTACTTGCTAT</td><td>CTCCTCGACTATGTTCCCATCA</td></tr>
<tr><td><a href="seqData/TGW6_3.3_3DL.txt">TGW6-3.3-3DL</a></td><td>WL3995/WL3996</td><td>TAACCAAGAGTCCAAGAGCCAAG</td><td>CAATAGTCCATCCGACGAAAAG</td></tr>
<tr><td><a href="seqData/TGW6_7.1_7AS.txt">TGW6-7.1-7AS</a></td><td>WL4015/WL4016</td><td>TGAATCCATCAAACCAAACCATA</td><td>TTCAATTCTGGGAAGGGAAGAAC</td></tr>
<tr><td><a href="seqData/TGW6_7.1_7DS.txt">TGW6-7.1-7DS</a></td><td>WL4017/WL4018</td><td>CGCGCCGGGTATAAACTT</td><td>CCTGTTGGTGAAGCGCAA</td></tr>
<tr><td><a href="seqData/TGW6_7.4_7AS.txt">TGW6-7.4-7AS</a></td><td>WL4145/WL4146</td><td>AGTGTCCATGCAAGTTTGAA</td><td>TGACCAGCCGAGCTTGTG</td></tr>
<tr><td><a href="seqData/TGW6_7.4_7BS.txt">TGW6-7.4-7BS</a></td><td>WL4110/WL4111</td><td>GACCAGGTCGAATCGGAACTAC</td><td>GATGACGAGATGTGTCCGA</td></tr>
<tr><td><a href="seqData/TGW6_7.4_7DS.txt">TGW6-7.4-7DS</a></td><td>WL4027/WL4028</td><td>AAAGAAACCATGAAAAGGAATCG</td><td>TTTCAAGCTCACACATCTGCT</td></tr>
</tbody>
</table></center>
<br><center><button class="btn"style="width:50%"><i class="fa fa-download"></i> Download Sequences</button></center><br>
</div>
<hr>

								<p>We have developed an Agrobacterium-delivered CRISPR/Cas9 system for wheat genome editing. We tested this system in wheat protoplasts using a GFP reporter assay, which includes a 1-bp insertion in GFP coding region that disrupted the protein function, wheat codon-optimized Cas9, and a guide RNA targeting the 1-bp insertion. The protoplast assay not only demonstrated functionality of the Cas9 but also selected U6.1 and U6.3 promoter for driving guide RNA cassettes. Using this CRISPR/Cas9 system, we developed 13 constructs for editing 11 grain-regulatory genes. So far we have identified 64 mutation in T0, T1 and T2 generations for TaCKX2-1, TaGLW7, TaGW2 and TaGW8. Analysis of the mutations showed that majority of them are due to deletions.

									We also developed a PCR-Cas9 RNP assay protocol for rapid and accurate detection of edit mutations.</p>

									<hr><button type="button" class="collapsible"><center>Constructs</center></button><div class="content"> 
										<br><center><button class="btn"style="width:50%"><i class="fa fa-download"></i> Download Constructs</button></center><br>
									</div>
									<hr>
							</div>
						</div>
					</div>
				</div>
				
				
			

				

			</main> <!-- .main-content -->
			<hr>
			<footer class="site-footer">
				<div class="container">
					<div class="row">
						<div class="col-md-3">
							<div class="widget">
								<h3 class="widget-title">Our address</h3>
								<strong>Dr Wanlong Li's Lab</strong>
								<address>McFadden Biostress Building SNP223
									South Dakota State University
									Brookings, SD 57007</address>
								<a href="tel:605-688-5743">605-688-5743</a> <br>
								<a href="wanlong.li@sdstate.edu">Email: wanlong.li@sdstate.edu</a>
							</div>
						</div>
						<div class="col-md-3">
							<div class="widget">
								<h3 class="widget-title">Research Grants</h3>
								
									<li><a href="#">USDA</a></li> 
									<li><a href="#">NSF</a></li>
									
								
							</div>
						</div>
						<div class="col-md-3">
							<div class="widget">
								<h3 class="widget-title">Research summary</h3>
							
									<li><a href="#">Rna seq</a></li> 
									<li><a href="#">transcriptome</a></li>
									<li><a href="#">genome</a></li> 
									<li><a href="#">snp</a></li>
									<li><a href="#">ssr</a></li>
								
							</div>
						</div>
						<div class="col-md-3">
							<div class="widget">
								<h3 class="widget-title">Social media</h3>
								<p>You can also contact us at.</p>
								<div class="social-links">
									<a href="#"><i class="fa fa-facebook"></i></a>
									<a href="#"><i class="fa fa-twitter"></i></a>
									<a href="#"><i class="fa fa-google-plus"></i></a>
									<a href="#"><i class="fa fa-pinterest"></i></a>
								</div>
							</div>
						</div>
					</div> <!-- .row -->

					
				</div> <!-- .container -->
			</footer>
		</div>

		<script src="js/jquery-1.11.1.min.js"></script>
		<script src="js/plugins.js"></script>
		<script src="js/app.js"></script>
		<script>
			var coll = document.getElementsByClassName("collapsible");
var i;

for (i = 0; i < coll.length; i++) {
  coll[i].addEventListener("click", function() {
    this.classList.toggle("active");
    var content = this.nextElementSibling;
    if (content.style.display === "block") {
      content.style.display = "none";
    } else {
      content.style.display = "block";
    }
  });
} 
		</script>
		
	</body>

</html>